code/modules/mob/living/carbon/life.dm

```diff
@@ -430,7 +430,9 @@ GLOBAL_LIST_INIT(ballmer_windows_me_msg, list("Yo man, what if, we like, uh, put
 												"Best idea ever: Disposal pipes instead of hallways.",
 												"We should store bank records in a webscale datastore, like /dev/null.",
 												"You ever wonder if /dev/null supports sharding?",
-												"What if we use a language that was written on a napkin and created over 1 weekend for all of our servers?"))
+												"What if we use a language that was written on a napkin and created over 1 weekend for all of our servers?",
+												"We should totally invent DNA-locked disablers with a red alert emitter mode.",
+												"Magical idea: Atmos pipes that only work 20% of the time!"))
```

Comment:
words shouldn't be capitalized after colons
would also like it to reflect the optimization purpose of it more as well

Reply:
> words shouldn't be capitalized after colons

This is only the case for introducing lists and in bad style guides.

Reply:
Reworded

Reply:
but atmos pipes that only work 20% of the time is a dependent clause